src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -3002,6 +3002,11 @@ theorem toInt_neg_of_ne_intMin {x : BitVec w} (rs : x ≠ intMin w) :
   have := @Nat.two_pow_pred_mul_two w (by omega)
   split <;> split <;> omega
 
+theorem toInt_neg_eq_ite {x : BitVec w} :
+    (-x).toInt = if x = intMin w then x.toInt else -(x.toInt) := by
+  by_cases hx : x = intMin w <;>
+    simp [hx, neg_intMin, toInt_neg_of_ne_intMin]
+
```

Comment:
Is this theorem unused in your PR? In case it is, you may not need to stack your PR.

Reply:
Right, I avoided using it.

Reply:
Maybe also remove the mention of stacking in your PR.